wysiwyg.js

```diff
@@ -0,0 +1,148 @@
+				  lifespan: {
+				    birth: 1757,
+				    death: 1854
+				  }
+				},
+					  {title: "Major General",
+				  name: "Marie-Joseph Paul Yves Roch Gilbert du Motier, Marquis de Lafayette ",
+				  bio: "A French aristocrat and military officer who fought in the American Revolutionary War. A close friend of George Washington, Alexander Hamilton, and Thomas Jefferson, Lafayette was a key figure in the French Revolution of 1789 and the July Revolution of 1830.",
+				  image: "https://www.biography.com/.image/c_fill,cs_srgb,dpr_1.0,g_face,h_300,q_80,w_300/MTIwNjA4NjM0MjM1OTQ2NTA4/marquis-de-lafayette-21271783-1-402.jpg",
+				  lifespan: {
+				    birth: 1757,
+				    death: 1834
+				  }
+				},
+					  {title: "Mister",
+				  name: "Hercules Mulligan",
+				  bio: "A tailor and spy during the American Revolutionary War.",
+				  image: "https://pixel.nymag.com/imgs/daily/vulture/2016/12/26/26-george-michael-1.w190.h190.2x.jpg",
+				  lifespan: {
+				    birth: 1740,
+				    death: 1825
+				  }
+				},
+					  {title: "King",
+				  name: "George William Frederick",
+				  bio: "King of Great Britain and Ireland until the union of the two countries on, after which he was King of the United Kingdom of Great Britain and Ireland until his death. He was concurrently Duke and prince-elector of Brunswick-Lüneburg in the Holy Roman Empire until his promotion to King of Hanover on 12 October 1814.",
+				  image: "https://www.biography.com/.image/c_fill,cs_srgb,dpr_1.0,g_face,h_300,q_80,w_300/MTMxNjM0MjYxMTQ4MDUyMDk5/king-george-iii-wikicommonsjpg.jpg",
+				  lifespan: {
+				    birth: 1760,
+				    death: 1820
+				  }
+				}
+					]
+
+				
+
+for (var i = 0; i < famousPeople.length; i++) {
+	var historyCard = "";
+  // historyCard += `<div class="person__container" id="person--${i}"></div>`;
+	historyCard += `<header>${famousPeople[i].title} ${famousPeople[i].name}</header>`
+	historyCard += `<section>${famousPeople[i].bio}<br> <img src=${famousPeople[i].image}></section>`
+	historyCard += `<footer>Born: ${famousPeople[i].lifespan.birth}<br>Died: ${famousPeople[i].lifespan.death} </footer>`
+	cardHolder.children[i].innerHTML = historyCard;
+	cardHolder.children[i].className += " person__container";
+	cardHolder.children[i].id += `person--${i}`;
+
+}
+
+// function whatWasThatClick(event) {
+// 	console.log(event);
+// 	console.log(event.target.children[0]);
+// }
+// document.body.addEventListener("click", whatWasThatClick);
+
+function personElementSelector(selectedEl) {
+	classSelected = document.getElementsByClassName("selected");
+			if (selectedEl.classList[2] === "selected") {
+			selectedEl.classList.toggle("selected");
+		} else {
+			while (classSelected.length)
+				classSelected[0].classList.remove("selected");
+			selectedEl.classList.toggle("selected");
+		}
+		if (selectedEl.classList[2] === "selected") {
+		userInputField.focus();
```

Comment:
You should be able to move these two lines targeting the userInputField within the if statement used earlier in this function, since the logical condition is the same. After you do that, you should be able to take out this if {} else {} statement